index.js

```diff
@@ -3,7 +3,7 @@
 var assign = require('object.assign');
 
 var modules = [require('babel-plugin-transform-es2015-modules-commonjs'), {
-  strict: false
+  strict: true
 }];
```

Comment:
I don't think so - we need to be able to support `modules` true vs false, so that the default is safe, but that we're able to turn off module transforms for webpack tree-shaking.

Reply:
babel-preset-env has this option too though. Here's the documentation (for a newer version I think) https://babeljs.io/docs/en/next/babel-preset-env.html#modules

Reply:
My point is that we are already using babel-preset-env and we might be able to get the same outcome as what you have here and clean up this module a little by removing lines 5-7 (https://github.com/airbnb/babel-preset-airbnb/blob/f1bbda2f0be71b37615df425f2f7fc317ce64bcf/index.js#L5-L7) and updating lines 38 (https://github.com/airbnb/babel-preset-airbnb/blob/f1bbda2f0be71b37615df425f2f7fc317ce64bcf/index.js#L38) 44 (https://github.com/airbnb/babel-preset-airbnb/blob/f1bbda2f0be71b37615df425f2f7fc317ce64bcf/index.js#L44)

Reply:
If that’s indeed identical, I’m happy to make the change, but i think enabling strict mode alone is important to get out.

How confident are you that would be identical?

Reply:
that does ring a bell.

I'm going to push forward with this, and we can explore switching fully to env in a future PR.